### ISSUE TITLE
vertical spacing too small

### ISSUE THREAD

**Issue by boustanihani** [djhi/meteor-autoform-materialize#33]
In some cases, the vertical spacing between the "input-fields" is very small, probably because not wrapping each input-field inside rows & columns... (Field-1 label almost touches the above line when a value is being entered)

Reproduction deployment: http://materialize-autoform.meteor.com
Repository: https://github.com/boustanihani/materialize-autoform

``` html
<body>
    <h5 class="center">Materialize AutoForm</h5>

    <div class="row">
        <div class="col s4 offset-s4">
            {{> hello}}
        </div>
    </div>
</body>

<template name="hello">
    {{> quickForm id="tableCreate" schema="Schema.test"}}
</template>
```

``` javascript
Schema.test = new SimpleSchema({

    size1: {
        type: Number,
        label: 'Size-1',
        allowedValues: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
    },

    size2: {
        type: Number,
        label: 'Size-2',
        allowedValues: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
        autoform: {
            firstOption: 'Choose Size-2',
            options: 'allowed',
        }
    },

    field1: {
        type: String,
        label: 'Field-1',
    },

    field2: {
        type: String,
        label: 'Field-2',
    },
});
```

**Comment by djhi** [comment]
Fixed in `0.0.20` which will be released today

**Comment by byennen** [comment]
:+1:
